src/web/services/feedback-sessions.service.ts

```diff
@@ -57,4 +57,14 @@ export class FeedbackSessionsService {
     };
     return this.httpRequestService.get('/sessions/ongoing', paramMap);
   }
+
+  /**
+   * Gets the amount of responses for a specific question in a feedback session.
+   */
+  getResponsesCountForQuestion(questionId: string): Observable<HasResponses> {
```

Comment:
Shouldn't think be in question service? Why it is still responsesCount?

`hasResponsesForQuestion()`?